libbeat/cmd/modules.go

```diff
@@ -42,7 +42,7 @@ func GenModulesCmd(name, version string, modulesFactory modulesManagerFactory) *
 
 // Instantiate a modules manager or die trying
 func getModules(name, version string, modulesFactory modulesManagerFactory) ModulesManager {
-	b, err := instance.NewBeat(name, version)
+	b, err := instance.NewBeat(name, name, version)
```

Comment:
Could we leave it empty by default, means if it is `instance.NewBeat(name, "", version)` it will use the name automtically?